### PR TITLE
Add intx as Hunter package

### PR DESCRIPTION
**Pull request by chfast**
It upgrades Hunter and enables it by default. The intx 0.2 is added as the Hunter package.

### CODE REVIEW

**Comment by axic** [comment]
Will we merge it this way or are we waiting for its inclusion to Hunter? Also any chance release 0.1.1 with the le/be change?

**Comment by axic** [comment]
Are we wainting for https://github.com/ruslo/hunter/pull/1846 to be merged before merging this?

**Comment by chfast** [comment]
Yes

**Comment by axic** [comment]
Why not update cable first?

**Comment by chfast** [comment]
> Why not update cable first

For what?

**Comment by axic** [comment]
It does some of that hunter upgrade & moving stuff, doesn't it?

**Comment by chfast** [comment]
> It does some of that hunter upgrade & moving stuff, doesn't it?

No it doesn't. It only upgraded HunterGate which is the bootstrap script that downloads the Hunter itself. Upgrading it is usually not required. I'm just doing it from time to time for sanity.

File: cmake/Hunter/init.cmake [hunk 1]
```diff
@@ -1,12 +1,10 @@
 
-if(EVMC_TESTING OR EVMC_TEST_TOOLS)
-    set(HUNTER_CONFIGURATION_TYPES Release
-        CACHE STRING "Build type of the Hunter packages")
+set(HUNTER_CONFIGURATION_TYPES Release
+    CACHE STRING "Build type of the Hunter packages")
 
-    include(HunterGate)
+include(HunterGate)
```
Comment (axic) [inline cmake/Hunter/init.cmake]: How is this link resolved?

Also can you move the whitespace changes to the first commit which introduces the file?

Comment (chfast) [reply cmake/Hunter/init.cmake]: It's in `cmake/cable` which is added to CMake's module path.

What whitespace changes?

Comment (axic) [reply cmake/Hunter/init.cmake]: The second commit removes spaces from the file added in the first commit.

Comment (chfast) [reply cmake/Hunter/init.cmake]: It removes `if()` so the indentation changes.

**Pull request merged**